frontend/packages/knative-plugin/src/topology/knative-topology-utils.ts

```diff
@@ -1159,6 +1159,14 @@ export const transformKnNodeData = (
   _.forEach(knResourcesData, (res) => {
     const item = createKnativeDeploymentItems(res, resources, utils);
     switch (type) {
+      case NodeType.KafkaSink: {
+        const data = createEventSinkTopologyNodeData(res, item, type);
+        const itemData = getOwnedEventSinkData(res, data, resources);
+        knDataModel.nodes.push(...getKnativeTopologyNodeItems(res, type, itemData, resources));
+        const newGroup = getTopologyGroupItems(res);
+        mergeGroup(newGroup, knDataModel.nodes);
+        break;
+      }
```

Comment:
can we have them in same case? adn i blv for `kafkaSink` `getEventSinkTopologyEdgeItems` would return `[]`. WDYT?
```suggestion
      case NodeType.EventSink:
      case NodeType.KafkaSink: {
        const data = createEventSinkTopologyNodeData(res, item, type);
        const itemData = getOwnedEventSinkData(res, data, resources);
        knDataModel.nodes.push(...getKnativeTopologyNodeItems(res, type, itemData, resources));
        const newGroup = getTopologyGroupItems(res);
        mergeGroup(newGroup, knDataModel.nodes);
        break;
      }
```